miniml/compiler/test/Makefile

```diff
@@ -16,22 +15,35 @@ make OCAMLRUN=../../../_boot/byterun/ocamlrun
 
 endef
 
-
+OCAMLRUN_CMD=$(or $(OCAMLRUN),$(error $(ERRMSG)))
 COMPILE_CMD=GUILE_WARN_DEPRECATED=detailed guile ../compile.scm
 COMPILE_DEPS=../compile.scm
 
+TESTS=\
+	arith \
+	functions \
+	patterns \
+	lists \
+	labels \
+	records \
+	exceptions \
+	let_open \
+	infix_sugar \
+	external_exceptions
```

Comment:
It seems like `functors` is missing in this list.

Reply:
Thanks! Fixed.